app/services/normalise_charge.js

```diff
@@ -155,7 +156,8 @@ module.exports = (function () {
       'prepaid': card.prepaid,
       'address': addressForApi(req.body),
       'accept_header': req.header('accept'),
-      'user_agent_header': req.header('user-agent')
+      'user_agent_header': req.header('user-agent'),
+      'ip_address': userIpAddress(req)
```

Comment:
I know this may be copied over from another service but I wonder if we could name `userIpAddress` more meaningfully. Especially as it's in the generic utils folder. 

It seems like it's used to extract the source ip address from the req object no matter where it is.

Reply:
`source_ip_address`? Suggestions?

Reply:
I don't have any suggestions here I just don't know what calling the function `userIpAddress` might do, especially as it's imported directly as a module. 

Some patters might look like 
`utils/IPParser` -> `IPParser.getIpFromRequest(req)` 

I didn't request changes on this review as it's not dissimilar to other code, I just think we need to think about our naming more carefully on our Node apps.

Reply:
Sure - I don't mind renaming. I just expect suggestions when I'm asked to do so :) (that's it)